### PR TITLE
fix: APPS-1879 title summary homepage, buttonUrl BlockCTA fragment

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-1879](https://jira.library.ucla.edu/browse/APPS-1879)

### CODE REVIEW

File: gql/fragments/BlockCallToActionFragment.gql [hunk 1]
```diff
@@ -8,7 +8,7 @@ fragment BlockCallToActionFragment on ElementInterface {
             summary
             icon
             buttonText
-            buttonLink
+            buttonUrl
```
Comment (pghorpade) [inline gql/fragments/BlockCallToActionFragment.gql]: buttonUrl: buttonLink

File: pages/index.vue [hunk 2]
```diff
@@ -5,7 +5,7 @@
     >
         <masthead-secondary
             title="Modern Endangered Archives Program"
```
Comment (pghorpade) [inline pages/index.vue]: pull title from craft too

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: pages/index.vue (hunk 2 above)
Comment (jendiamond) [reply pages/index.vue]: The Title will need to be update in craft then because it is currently not Modern Endangered Archives Program

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**